docs/handout_en.md

```diff
@@ -24,30 +24,30 @@ myslíme tím, že používáme linuxovou distribuci. Linuxová distribuce je
 kombinace linuxového jádra, grafického rozhraní a předinstalovaných
 aplikací. Dost teorie, pojďme na to.
 
-## Linux a nevidomí uživatelé
+## Linux and visually impaired users
 
-Protože tento workshop byl vytvořen pro začátečníky, předpokládám, že
-budete Linux používat skrze jeho grafické rozhraní. Jen pro úplnost
-dodávám, že jej lze ovládat i skrze čistě textové rozhraní.
+This workshop is aimed at beginner users, so I expect most of you
+will use Linux with the graphical user interface. However, note
+that you can fully control Linux by only using a text interface as well.
 
-Pro Linux existuje pouze jeden odečítač obrazovky, který se jmenuje
-[Orca](https://help.gnome.org/users/orca/stable/). Pro úplnost,
-odečítačů existuje více, ale jen tento lze použít s grafickým rozhraním.
-Ostatní odečítače se dají použít jen v příkazové řádce a tou se zde
-zabývat nebudeme.
+[Orca](https://help.gnome.org/users/orca/stable/) is the only
+screen reader that works completely on Linux. You can use other
+screen readers on Linux, but only Orca works with the graphical interface.
+Other screen readers work only in the command line interface, which
+is not in scope of this workshop.
 
-Budeme používat distribuci Fegora. Název Fegora vznikl spojením slov
-Fedora a Agora. [Fedora](https://mojefedora.cz/) je Linuxová distribuce
-podporovaná firmou [Red Hat](https://redhat.com/) a
+The subject of this workshop is Fegora. The name Fegora a contraction
```

Comment:
```suggestion
The subject of this workshop is Fegora. The name Fegora a conjunction
```

Reply:
I used `contraction` based on [this](https://en.wikipedia.org/wiki/Smog) article (that uses contraction). I think `blend` is the best; WDYT? 

What we want to say is `portmanteau` but IMHO many readers won't know what that means.

```suggestion
The subject of this workshop is Fegora. The name Fegora a blend
```